src/hexview/hexView.ts

```diff
@@ -218,42 +199,52 @@ export class DebuggerHexView {
             return;
         }
 
-        if (!this.arrowIconFileCreated) {
-            await this.createArrowIconFile();
-            this.arrowIconFileCreated = true;
-        }
-
-        if (!this.workspaceConfUpdated) {
-            this.updateWorkbenchConfig(false);
-            this.workspaceConfUpdated = true;
-        }
-
         if (this.dataFile === "") {
             await this.setDataFile();
         }
 
         let file = fs.readFileSync(this.dataFile);
         let hex = hexy.hexy(file);
+        let hexLines = hex.split("\n");
+        let formattedHex = "";
+
+        // Format hex code to make the file look nicer
+        hexLines.forEach(h => {
+            if (h) {
+                let splitHex = h.split(":");
+                let dataLocations = splitHex[1].split(" ");
+
+                formattedHex += splitHex[0] + ": ";
+                for(var i = 1; i < dataLocations.length-2; i++) {
+                    let middle = Math.floor(dataLocations[i].length/2);
+                    formattedHex += dataLocations[i].substr(0, middle).toUpperCase() + " " + dataLocations[i].substr(middle).toUpperCase() + " ";
+                }
+                formattedHex += "\t" + dataLocations[dataLocations.length-1] + "\n";
+            }
+        });
 
         // Create file that holds path to data file used
         if (!fs.existsSync(this.hexFile)) {
-            await fs.writeFile(this.hexFile, hex, function(err){
+            await fs.writeFile(this.hexFile, formattedHex, function(err){
                 if (err) {
                     vscode.window.showInformationMessage(`error code: ${err.code} - ${err.message}`);
                 }
             });
         }
 
-        // Open up hex document
-        if (!this.checkIfHexFileOpened()) {
-            this.openHexFile(body, hex);
-        }
-        else {
-            this.updateSelectedDataPosition(body, hex);
+        // Create arrow file and open up hex document only on start of debug
+        if (this.initialState) {
+            await this.createArrowIconFile();
```

Comment:
Why is the svg being created from code? Couldnt the arrow file from the vsix just be copied or even using in place from the extension?